python/python/Ice/Future.py

```diff
@@ -19,7 +19,7 @@ def __await__(self):
 
 def wrap_future(future, *, loop=None):
     """
-    Wrap an Ice.Future object into an asyncio.Future.
+    Wrap an :class:`Ice.Future` object into an ``asyncio.Future``.
```

Comment:
this `:class:` shows how to cross-reference see also https://www.sphinx-doc.org/en/master/usage/domains/python.html

Reply:
I think it works both ways, at least when using in a Python file.